cmake/AddChmTarget.cmake

```diff
@@ -88,14 +87,14 @@ function (add_chm_target docname lang entities figures dtd_files)
 
 
 
-    add_custom_target("${lang}-${docname}-chm"
+    add_custom_target("${lang}-${docname}-${fmt}"
```

Comment:
I'm in favour of keeping the hardcoded format names for reasones already mentioned in an earlier comment.